lib/jekyll/liquid_renderer/table.rb

```diff
@@ -75,18 +80,23 @@ def data_for_table(num_of_rows)
         sorted = @stats.sort_by { |_, file_stats| -file_stats[:time] }
         sorted = sorted.slice(0, num_of_rows)
 
-        table = [%w(Filename Count Bytes Time)]
+        gauges = [:count, :bytes, :time]
+        labels = %w(Filename).concat(gauges.map { |gauge| gauge.to_s.capitalize })
+        table  = [labels]
+        total  = Hash.new { |hash, key| hash[key] = 0 }
 
         sorted.each do |filename, file_stats|
-          row = []
-          row << filename
-          row << file_stats[:count].to_s
-          row << format_bytes(file_stats[:bytes])
-          row << format("%.3f", file_stats[:time])
-          table << row
+          gauges.each { |gauge| total[gauge] += file_stats[gauge] }
+          table << data_to_row(file_stats, [filename])
```

Comment:
this inline creation of the row doesn't make a lot of sense to me. There's a lot of intent hiding here that seems like it would be better to make explicit:
```
row = [filename]
table << add_data(file_stats, row)
```

Reply:
Can I use kw_args here to avoid lvar allocation?
```ruby
table << add_data_to_row(:data => file_stats, :row => [filename])
```

Reply:
I don't care about lvar allocation. I care about having readable code, particularly in pieces of code that don't need to be incredibly performant. The keyword arguments are better but they are still hiding intent. In fact, considering the amount of duplication is here (which is very little), I'd be just as happy to not have the method at all and just create the extra row for totals manually like was previously done in `master` as part of the loop iteration. Why? It's direct, straightforward, and incredibly explicit about what's going on. There's a tiny bit of code duplication, yes, but it's not a violation of the DRY principle or anything else.

Reply:
The abstraction to a new method was to appease RuboCop's ABC metrics cop

Reply:
Please disable that metric for this block of code.